sycl/test/basic_tests/aspects.cpp

```diff
@@ -0,0 +1,72 @@
+// RUN: %clangxx %s -o %t.out -I %sycl_include -lsycl
+// RUN: %t.out
+
+//==--------------- aspects.cpp - SYCL device test ------------------------==//
+//
+// Returns the various aspects of a device  and platform.
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+
+// platform::has() calls device::has() for each device on the platform.
+
+int main() {
+  bool failed = false;
+  int pltIdx = 0;
+  for (const auto &plt : platform::get_platforms()) {
+    pltIdx++;
+    if (plt.has(aspect::host)) {
+      std::cout << "Platform #" << pltIdx
```

Comment:
Currently this test only checks that has() method can be called, but it doesn't check its result. Probably it is better to query same info using existing SYCL methods (like get_info) and compare it with the result of has().

Reply:
The aspect::has() function calls get_info() to see if the requested aspect is supported.  Whether it is right or wrong, has() and get_info() will always agree.